src/App.css

```
@@ -0,0 +1,118 @@
+@font-face {
+  font-family: 'Gatwick';
+  src: url('./fonts/Gatwick-Bold.otf');
+}
+
+h1 {
+  text-align: center;
+  font-family: Gatwick;
+  letter-spacing: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+body, input, button {
+  font-family: Helvetica;
```

consider adding a full font stack, to provide guidance to the browser in case it doesn't support Helvetica (what browser doesn't support Helvetica I don't know)
```suggestion
  font-family: Helvetica Neue, Helvetica, Arial, sans-serif; 
```